core/src/main/java/org/apache/carbondata/core/scan/result/vector/CarbonColumnVector.java

```diff
@@ -116,6 +117,13 @@
 
   void setLazyPage(LazyPageLoader lazyPage);
 
+  default int getIntsSize() {
+    return CarbonV3DataFormatConstants.NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE_DEFAULT;
```

Comment:
please remove from here and keep only in impl. because this guy is not aware about Int array

Reply:
done